fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1,299 @@
+
```

Comment:
Can we copy this over on the demand, so that we change the fastlane rules, it will use the latest rules?